apple/internal/aspects/swift_static_framework_aspect.bzl

```diff
@@ -122,14 +132,15 @@ swift_library dependency with no transitive swift_library dependencies.\
             # Collect the interface artifacts. Only get the first element from each depset since
             # they should only contain 1. If there are transitive swift_library dependencies, this
             # aspect would have errored out before.
+            # TODO: This has to be fixed before merging
```

Comment:
this would need a `direct_generated_headers` counterpart I believe, if this approach makes sense